test/integration/volumescheduling/util.go

```diff
@@ -123,6 +123,13 @@ func initTestSchedulerWithOptions(
 
 	// set DisablePreemption option
 	context.schedulerConfig.DisablePreemption = false
+	eventBroadcaster := events.NewBroadcaster(&events.EventSinkImpl{
```

Comment:
why are you moving these lines?

Reply:
initializing the recorder after the call to `NewFromConfig`  didn't update in `sched.Recorder`. Any better idea to do this?

Reply:
SG